client/tests.json

```diff
@@ -33,6 +33,30 @@
 			},
 			"test": [ "assembler", "index", "store" ]
 		},
+		"reader-comment-email-subscriptions": {
```

Comment:
I started renaming all test files to mirror file under test and reword suite description to have the same name as file. That way we can easily find failing tests because test output directly translates into path. Let me know what do you think about this idea. We definitely can skip that in this PR :)